test/enrollment_test.rb

```diff
@@ -58,4 +58,16 @@ def test_sanitizes_participation
 
     assert_equal expected, sanitized
   end
+
+  def test_gets_average_participation_for_all_years
+    e = Enrollment.new({:name => "ACADEMY 20",
```

Comment:
Redundant curly braces around a hash parameter.<br>Use the new Ruby 1.9 hash syntax.<br>Space inside { missing.